engine/tests/mechanics/games/test_credentials_roster.py

```
@@ -0,0 +1,149 @@
+    Region,
+    Restrictions,
+    RestrictionLevel,
+    ScenarioOffer,
+    ShiftSpec,
+    derive_disposition,
+    generate_roster,
+    materialize,
+)
+
+D = CredentialDisposition
+I = Indication
```

Ruff (0.15.13)

[error] Ambiguous variable name: `I`

(E741)